### PR TITLE
Update to Server 5.1

### PR DESCRIPTION
**Pull request by robsdedude**

### CODE REVIEW

**Review by bigmontz — APPROVED** [review]
📖

File: modules/ROOT/pages/bolt/message.adoc [hunk 1]
```diff
@@ -399,13 +399,19 @@ The `routing::Dictionary(address::String)` field is added in version *4.1* and a
...
   user_agent::String,
+  patch_bolt::List<String>,
   scheme::String,
   routing::Dictionary(address::String),
   …
 )
 ----
 
 ** The `user_agent` should conform to `"Name/Version"` for example `"Example/4.1.0"` (see link:https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent[] for more information).
+** `patch_bolt` is version *4.4 only*.
```
Comment (thelonelyvulpes) [inline modules/ROOT/pages/bolt/message.adoc:410]: it's 4.3 and 4.4

Comment (bigmontz) [reply modules/ROOT/pages/bolt/message.adoc:410]: Was it ported to 4.3 servers?

Comment (robsdedude) [reply modules/ROOT/pages/bolt/message.adoc:410]: Checked the driver code: 4.3. Don't know about server @gjmwoods ?

Comment (gjmwoods) [reply modules/ROOT/pages/bolt/message.adoc:410]: yes ported back to 4.3

Comment (robsdedude) [inline modules/ROOT/pages/bolt/message.adoc:410]:
```suggestion
** `patch_bolt` is version *4.3-4.4 only*.
```

File: modules/ROOT/pages/bolt/message.adoc [hunk 2]
```diff
@@ -451,6 +457,7 @@ The following fields are defined for inclusion in the `SUCCESS` metadata:
 
 ** `server::String` (server agent string, example `"Neo4j/4.1.0"`)
 ** `connection_id::String` (unique identifier of the bolt connection used on the server side, example: `"bolt-61"`)
+** `patch_bolt` only in version *4.4* and only if the client requested patches in the `patch_bolt` field of the request. The server will include the subset of requested patches (with the exact same string the client requests) if it supports it. From that point onward, the server-client communication _must only_ use the patched protocol.
```
Comment (robsdedude) [inline modules/ROOT/pages/bolt/message.adoc:460]:
```suggestion
** `patch_bolt` only in versions *4.3* and *4.4* and only if the client requested patches in the `patch_bolt` field of the request. The server will include the subset of requested patches (with the exact same string the client requests) if it supports it. From that point onward, the server-client communication _must only_ use the patched protocol.
```

**Pull request merged**